### PR TITLE
Add Facebook Open Graph and Twitter Cards metadata

### PR DESCRIPTION
**Pull request by lucperkins**
This PR adds some basic Open Graph and Twitter Cards metadata to each page. I make a distinction here between the index page and documents (i.e. material in `/docs`). Metadata that applies to all pages is templated in `layouts/partials/meta.ace` and supplied via the `params` in `config.yaml`; metadata that only applies to the index page is templated in `layouts/partials/index/index-meta.ace`; and metadata for documents is templated in `layouts/partials/docs/page-meta.ace`.

### CODE REVIEW

**Comment by objmagic** [comment]
+1

**Comment by kramasamy** [comment]
@billonahill - do you have any comments on this?

File: website/config.yaml [hunk 1]
```diff
@@ -18,7 +18,8 @@ PygmentsCodeFences: true
 
 # Site-level parameters available through .Site.Params.{key}
 params:
-  author: Luc Perkins
+  author: Twitter, Inc.
+  description: A realtime, distributed, fault-tolerant stream processing engine from Twitter
   versions:
     heron: 0.13.7
```
Comment (kramasamy) [inline website/config.yaml]: @lucperkins - why the heron version is hard coded. It might be hard to maintain this.

**Comment by billonahill** [comment]
I'm not that familiar with the syntax of either but these changes seem reasonable.

**Pull request merged**